stdlib/builtins.pyi

```diff
@@ -95,9 +94,9 @@ _SupportsAnextT = TypeVar("_SupportsAnextT", bound=SupportsAnext[Any], covariant
 _AwaitableT = TypeVar("_AwaitableT", bound=Awaitable[Any])
 _AwaitableT_co = TypeVar("_AwaitableT_co", bound=Awaitable[Any], covariant=True)
 _P = ParamSpec("_P")
-_StartT = TypeVar("_StartT", covariant=True, default=Any)
-_StopT = TypeVar("_StopT", covariant=True, default=Any)
-_StepT = TypeVar("_StepT", covariant=True, default=Any)
+_StartT_co = TypeVar("_StartT_co", covariant=True, default=Any)
+_StopT_co = TypeVar("_StopT_co", covariant=True, default=_StartT_co)  #  slice[A] -> slice[A, A, Any]
```

Comment:
They are somewhat different, but as I said in the OP I think they are coupled, because when someone annotates something as, say `slice[datetime]`, they have certain expectations which kind of slice constructions will satisfy this type hint.

Reply:
Right. I'm just wondering if there's a _chance_ that the primer hit might go away if we got rid of this change. I can see that ideally we'd make the changes together, but I'd be curious to see a version of the PR that just does the constructor overload changes, so we can evaluate exactly which change it is that's causing the new error.

Reply:
https://github.com/python/typeshed/pull/12904 works without any hits.